python/mspasspy/util/converter.py

```diff
@@ -674,12 +680,12 @@ def Textfile2Dataframe(
     if (
         header_line is None or header_line < 0
     ):  #   Header_line not given, using attribute_names
-        if parallel:
+        if parallel and __mspasspy_has_dask:
             df = daskdf.read_csv(filename, sep=separator, names=attribute_names)
```

Comment:
Let's add a warning here. If parallel is specified but dask doesn't exist, the user need to be aware of the fact that it is running pandas dataframe. 

Same with the rest below.